src/date-range-picker/index.tsx

```diff
@@ -27,6 +27,7 @@ import { warnOnce } from '../internal/logging.js';
 import { usePrevious } from '../internal/hooks/use-previous/index.js';
 import { useUniqueId } from '../internal/hooks/use-unique-id';
 import { formatTimezoneOffset, getBrowserTimezoneOffset, isIsoDateOnly } from '../internal/utils/date-time';
+import { formatValue } from './use-date-range-picker.js';
```

Comment:
Any reason this function is in the hook file and not in the utils/date-time file like the other formatters on the line above?

Reply:
This function is somewhat specific to the range picker since it formats the value depending on the range type so I would think it would make sense to keep it within the hook?. The above functions seem to all be quite generic which is why I think they're in the utils folder

Reply:
I think this can be tackled as a follow-up